tests/src/test/java/com/example/SilencedFuzzer.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueHigh;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintStream;
+
+public class SilencedFuzzer {
+  private static final PrintStream noopStream = new PrintStream(new OutputStream() {
+    @Override
+    public void write(int b) {}
+  });
+
+  public static void fuzzerInitialize() {
+    System.setErr(noopStream);
+    System.setOut(noopStream);
+  }
+
+  public static void fuzzerTestOneInput(byte[] input) {
+    System.err.println("very noisy");
+    System.out.println("even more noise");
+    throw new FuzzerSecurityIssueHigh();
```

Comment:
What does this test verify? If it's used to manually check the output, we should mark it as such in Bazel.

Reply:
Added a comment explaining how the test is supposed to work and removed the unnecessary output.

Reply:
Ah, now I understand the idea of this test. Thx.